tests/s3/versioning/test_delete_object_tagging.py

```diff
@@ -348,3 +348,170 @@ def test_del_obj_tag_40436(self):
+                    self.object_name)
+        resp = s3_ver_tlib.get_object_tagging(s3_tag_test_obj=self.s3_tag_obj,
+                                              s3_ver_test_obj=self.s3_ver_obj,
+                                              bucket_name=self.bucket_name,
+                                              object_name=self.object_name)
+        assert_utils.assert_true(resp[0], resp)
+        # expecting "TagSet": []
+        assert_utils.assert_false(resp[1], resp)
+
+        LOGGER.info("Step 13: DELETE Object Tagging for %s (with versionId=null) %s", self.object_name,
+                    ver_list[1])
+        resp = s3_ver_tlib.delete_object_tagging(bucket_name=self.bucket_name,
+                                                 s3_ver_test_obj=self.s3_ver_obj,
+                                                 s3_tag_test_obj=self.s3_tag_obj,
+                                                 object_name=self.object_name,
+                                                 version_id=ver_list[1])
+        assert_utils.assert_true(resp[0], resp)
+
+        LOGGER.info("Step 14: GET Object Tagging for %s with versionId=%s", self.object_name,
+                    ver_list[0])
+        resp = s3_ver_tlib.get_object_tagging(s3_tag_test_obj=self.s3_tag_obj,
+                                              s3_ver_test_obj=self.s3_ver_obj,
+                                              bucket_name=self.bucket_name,
+                                              object_name=self.object_name,
+                                              version_id=ver_list[0])
+        assert_utils.assert_true(resp[0], resp)
+        # expecting "TagSet": []
+        assert_utils.assert_false(resp[1], resp)
+
+        LOGGER.info("Step 15: GET Object Tagging for %s (with versionId=null)", self.object_name)
+        resp = s3_ver_tlib.get_object_tagging(s3_tag_test_obj=self.s3_tag_obj,
+                                              s3_ver_test_obj=self.s3_ver_obj,
+                                              bucket_name=self.bucket_name,
+                                              object_name=self.object_name,
+                                              version_id=ver_list[1])
+        assert_utils.assert_true(resp[0], resp)
+        # expecting "TagSet": []
+        assert_utils.assert_false(resp[1], resp)
+
+        LOGGER.info("Step 16: GET Object Tagging for %s (without versionId specified)",
+                    self.object_name)
+        resp = s3_ver_tlib.get_object_tagging(s3_tag_test_obj=self.s3_tag_obj,
+                                              s3_ver_test_obj=self.s3_ver_obj,
+                                              bucket_name=self.bucket_name,
+                                              object_name=self.object_name)
+        assert_utils.assert_true(resp[0], resp)
+        # expecting "TagSet": []
+        assert_utils.assert_false(resp[1], resp)
```

Comment:
Didn't get this validation?

Reply:
resp['TagSet'] 
resp[1] is a TagSet (list from a resp dictionary ) from libs.s3.s3_versioning_test_lib.S3VersioningTestLib.get_obj_tag_ver
which we are expecting as empty.